exchanges/bybit/bybit_test.go

```diff
@@ -838,8 +835,8 @@ func TestPlaceOrder(t *testing.T) {
 		Side:      "buy",
 		OrderType: "limit",
 	})
-	if !errors.Is(err, order.ErrAmountBelowMin) {
-		t.Fatalf("expected %v, got %v", order.ErrAmountBelowMin, err)
+	if !errors.Is(err, limits.ErrAmountBelowMin) {
```

Comment:
```suggestion
	require.ErrorIs(t, err, limits.ErrAmountBelowMin)
```
And same for the below ones